src/DynamoCore/Configuration/LuceneConfig.cs

```diff
@@ -138,7 +143,8 @@ public enum IndexFieldsEnum
                                                    nameof(IndexFieldsEnum.Description),
```

Comment:
Change the Enum name from IndexFieldsEnum as NodesfieldsEnum as it is specific to that?

Reply:
agreed

Reply:
I've renamed IndexFieldsEnum to NodeIndexFields
commit: https://github.com/DynamoDS/Dynamo/pull/14136/commits/fab8b0751ea15150204c248692520e067db5ae91

Reply:
Sorry for the confusion, I just saw that the same enum property is used in PackageIndexFields also and one of the enum values is "Hosts". Not a big deal, you can revert this last name change if you want.